src/dispatchers/mouseDispatcher.ts

```diff
@@ -116,14 +123,31 @@ export module Dispatcher {
     }
 
     /**
+     * Registers a callback to be called whenever a wheel occurs,
+     * or removes the callback if `null` is passed as the callback.
+     *
+     * @param {any} key The key associated with the callback.
+     *                  Key uniqueness is determined by deep equality.
+     * @param {WheelCallback} callback A callback that takes the amount of mouse scroll
+     *                                     and the pixel position in svg-coordinate-space.
+     *                                     Pass `null` to remove a callback.
+     * @return {Dispatcher.Mouse} The calling Dispatcher.Mouse.
+     */
+    public onWheel(key: any, callback: WheelCallback): Dispatcher.Mouse {
+      this._setCallback(this._wheelBroadcaster, key, callback);
+      return this;
+    }
+
+    /**
      * Computes the mouse position from the given event, and if successful
      * calls broadcast() on the supplied Broadcaster.
      */
-    private _measureAndBroadcast(e: MouseEvent, b: Core.Broadcaster<Dispatcher.Mouse>) {
+    private _measureAndBroadcast(e: MouseEvent, b: Core.Broadcaster<Dispatcher.Mouse>,
+                                 otherBroadcastData: any[] = []) {
       var newMousePosition = this.translator.computePosition(e.clientX, e.clientY);
       if (newMousePosition != null) {
         this._lastMousePosition = newMousePosition;
-        b.broadcast(this.getLastMousePosition(), e);
+        b.broadcast.apply(b, otherBroadcastData.concat(this.getLastMousePosition(), e));
```

Comment:
The old implementation made the assumption that it will only ever broadcast the mousePosition, but there has to be some adjustment in order to broadcast any other information (like the changes in the deltaY of the mousewheel)